src/pages/Miruro/main.ts

```diff
@@ -0,0 +1,75 @@
+import { pageInterface } from '../pageInterface';
+
+export const Miruro: pageInterface = {
+  name: 'Miruro',
+  domain: ['https://www.miruro.tv', 'https://www.miruro.online'],
+  languages: ['English'],
+  type: 'anime',
+  isSyncPage(url) {
+    return Boolean(j.$('.player[data-media-player]').length);
+  },
+  isOverviewPage(url) {
+    return false;
+  },
+  getImage() {
+    return $('.player[data-media-player] .vds-poster').attr('src');
+  },
+  sync: {
+    getTitle(url) {
+      return utils.urlPart(url, 5).replace(/-/g, ' ').trim();
+    },
+    getIdentifier(url) {
+      // '139630-1' for https://www.miruro.tv/watch/139630/boku-no-hero-academia-6th-season/1
+      return `${utils.urlPart(url, 4)}-${utils.urlPart(url, 6)}`;
+    },
+    getOverviewUrl(url) {
+      const href = `https://${window.location.hostname}/watch/${utils.urlPart(url, 4)}`;
+      if (typeof href !== 'undefined') {
+        return utils.absoluteLink(href, Miruro.domain);
+      }
+      return '';
+    },
+    getEpisode(url) {
+      const temp = utils.urlPart(url, 6);
+      if (!temp) return NaN;
+      return Number(temp);
+    },
+    nextEpUrl(url) {
+      const href = `https://${window.location.hostname}/watch/${utils.urlPart(url, 4)}/${utils.urlPart(url, 5)}/${Miruro.sync.getEpisode(url) + 1}`;
+      if (typeof href !== 'undefined') {
+        return utils.absoluteLink(href, Miruro.domain);
+      }
+      return '';
+    },
+    getMalUrl(provider) {
```

Comment:
Why not use the buttons here. This way the mal url can be passed too. 
![image](https://github.com/MALSync/MALSync/assets/12599334/3ccdcafb-2660-4e22-a0a7-a4a01749962a)